_docs/_latest/concepts/best-practices.md

```diff
@@ -40,6 +40,135 @@ Scanning for external identities helps stop untrusted users from getting access
 When you add external users directly to a Cloud IAM policy, it's easier to audit their activities
 and do lifecycle management.
 
+**User Account Strategy**
+
+Ensure all employees are using a [Google Account from the enterprises Cloud
+Identity](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#use_corporate_login_credentials).
+This gives enterprise administrators full control over the lifecycle and
+security policies of that user account; deprovision when user leaves, enforce
+2SV, etc.
+
+**Project Naming Convention**
```

Comment:
@blueandgold Is there a way we can check if a rule is NOT implemented? 
Forseti could probably not check the quality of the naming convention, but could check if there's one at all?